### PR TITLE
Updated player mappings.

### PR DESCRIPTION
**Pull request by TheMathNinja**
Ok I followed all the steps! Let me know if this works. Not sure if this file really needs the blank line at the end still. Not sure how to add that; hopefully it works without a blank line.

### CODE REVIEW

**Comment by mrcaseb** [comment]
My commit doesn't fix the problem because it applies only for multiple files. We need to fix the attributes in parquet files in the releases

**Comment by mrcaseb** [comment]
Parquet problems in tests are fixed. This PR is up to you now, @tanho63

File: data-raw/clean_player_names.csv [hunk 1]
```diff
@@ -109,4 +109,12 @@ Janoris Jenkins,Jackrabbit Jenkins
...
\ No newline at end of file
+Tank Dell,Nathaniel Dell
+Julius Brents,JuJu Brents
+Tariq Woolen,Riq Woolen
+Jartavius Martin,Quan Martin
+Marquise Brown,Hollywood Brown
+YaYa Diaby,Yaya Diaby
+Michael Danna,Mike Danna
+Ogbonnia Okoronkwo,Ogbo Okoronkwo
+Henry To'oTo'o,Henry TooToo
```
Comment (tanho63) [inline data-raw/clean_player_names.csv:120]: I don’t think there is a scenario where clean names will keep these apostrophes, it is hardcoded to remove them. That probably makes the entry redundant.

**Review by tanho63 — COMMENTED** [review]
Remainder of this looks good. Can you add a line to NEWS.md describing the changes you made?

**Comment by TheMathNinja** [comment]
Yes but also a quick question. This was an immediate fix I submitted for an auction that’s now over. Now that I have proof of concept in submitting the PR would it be better to submit just this now or submit a more full one with a complete mapping fix? Or should I do a second one for the complete fix?

**Comment by tanho63** [comment]
submit now and then PR the remainder after if you want

**Review by tanho63 — APPROVED** [review]

**Pull request merged**